### ISSUE TITLE
Qeta interface get broken after upgrade

### ISSUE THREAD

**Issue by levanvunam** [drodil/backstage-plugin-qeta#244]
Firstly thanks for the awesome plugin.

I'm having issue with qeta after upgrade to: 3.12.3

![image](https://github.com/user-attachments/assets/6a353c23-e669-4c63-99e3-4f082cde01c4)

I see that the html generated in browser now is already have v5 prefix

![image](https://github.com/user-attachments/assets/e5c1dfb2-219e-4aee-aeea-187fb506507f)

compare to older version I had in my dev environment

![image](https://github.com/user-attachments/assets/f658172d-e0d3-4071-848a-f256f3fa67b8)

It seems to be because of MuiV5 (not very sure), but I already doing the mui5 update using: https://backstage.io/docs/tutorials/migrate-to-mui5/

Not sure anything is missing here

Thanks for helping

**Comment by drodil** [comment]
It seems the MUI5 with custom theme breaks everything. Here's a discussion about this; https://github.com/drodil/backstage-plugin-qeta/discussions/242

Probably have to rollback to MUI4 unless I can figure out how to make this work. Any ideas are more than welcome, I feel like I am in dead end soon 😄

**Comment by levanvunam** [comment]
haha thanks. Waiting for your final decision ;)

**Comment by drodil** [comment]
I tried to create a custom theme to a plain backstage instance (by following this https://backstage.io/docs/getting-started/app-custom-theme/#creating-a-custom-theme) and the Q&A works just fine:

![image](https://github.com/user-attachments/assets/382fc81c-ad87-4ee1-ace8-23847e36de9f)

I really don't know what is going on in our instance or with yours @levanvunam and @ahmm16 .. Still need to dig deeper!

**Comment by drodil** [comment]
At least the `Grid` component is not styled properly and is missing padding, for example. I have no clue where this comes from, though, as it works with a custom theme in the plain backstage instance.

**Comment by drodil** [comment]
At least the menu items in the plugin are messed with the `MuiButtonBase-root` CSS class that is applied over everything else:

![image](https://github.com/user-attachments/assets/575a55d8-a7f6-413f-8d4c-f01867afc965)

Disabling the styling makes the menu look like it did previously. But I have no clue why that is applied as last and it overrides everything. I mean, yeah it could be as base class for the buttons but now it's applied over everything else.

**Comment by drodil** [comment]
Hm, some of the issues were fixed by adding "@mui/material": "5.16.7" to package.json resolutions. @levanvunam could you try that?

**Comment by levanvunam** [comment]
Yes I'm having problem to downgrade to 3.9.3 due to the dependency (react, common) always go latest 3.12.3 now, making it not working

Let me try again with the package you mentioned

**Comment by drodil** [comment]
Yeah, downgrade also requires use of resolutions :/

**Comment by levanvunam** [comment]
After checking seems I already use 5.16.7 now for all mui related packages
![image](https://github.com/user-attachments/assets/2ccf0e3f-fd95-43c3-ac13-a51bbf3552f2)

I can still see the error as previously

![image](https://github.com/user-attachments/assets/5e46508a-467b-4d0a-a44a-e215daf06a67)

I try to look at those that I used for Grid, after migrate to mui5, I use Grid 2 and working ok
![image](https://github.com/user-attachments/assets/00d54209-cd53-412e-bf92-faa02cfdc0e7)

But when use Grid instead, the padding seems already a problem

![image](https://github.com/user-attachments/assets/c879bb83-667e-4594-838d-beeb234fe122)

Because of this: https://mui.com/material-ui/migration/migration-grid-v2/#with-material-ui-v5

So I'm not sure how you make the Grid working in your case when migrate to muiv5

**Comment by drodil** [comment]
Did you try to put it in the resolutions as follows:

```json
"resolutions": {
  "@mui/material": "5.16.7"
}
```

For example Backstage itself has "^5.12.2" so the version might be resolved to something different. For us, the resolution worked and fixed both the menu and the grid. 

It would make sense to move to Grid2 at some point as the current grid is going to be deprecated in MUI6.

**Comment by levanvunam** [comment]
Let me try and update again

**Comment by levanvunam** [comment]
I tried to downgrade the "@mui/material": "5.16.7", but seems no help.

I end up downgrage the package using resolutions for now :'(

**Comment by drodil** [comment]
There's this RFC in Backstage https://github.com/backstage/backstage/issues/27726 ... Probably best idea is to rollback this plugin to MUI4 and follow that... Will work on this soonish!

**Comment by drodil** [comment]
Reverted back to MUI4 and will publish a new 3.13.0 release soonish!
